lighthouse-core/report/html/templates.html

```diff
@@ -598,6 +598,31 @@
         animation-delay: 250ms;
     }
 
+    .lh-gauge__svg-wrapper {
+      position: relative;
+    }
+    .lh-gauge__wrapper--plugin .lh-gauge__svg-wrapper:before {
```

Comment:
add comment above

    /* The plugin badge overlay */

Reply:
done